playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,43 @@
+filebeat:
+  # List of prospectors to fetch data.
+  prospectors:
+    # Each - is a prospector. Below are the prospector specific configurations
+    -
+      paths:
+        - /var/log/syslog
+        - "/var/log/mail.*"
+        - "/var/log/kern.*"
+        - "/var/log/news.*"
+        - "/var/log/*.emerg"
+        - "{{ elasticsearch_log_dir }}/*"
+        - "{{ nginx_log_dir }}/*.log"
+        - "{{ nginx_log_dir }}/*.err"
+        - "{{ COMMON_LOG_DIR }}/tracking/*.log"
+        - "{{ COMMON_LOG_DIR }}/cms/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/util1/xqueue/edx.log"
+        - "{{ COMMON_LOG_DIR }}/devpi/supervisor/*"
+        - "/edx/app/edxapp/edx-platform/logs/util1/supervisor/*"
+        - "/edx/app/edxapp/edx-platform/logs/util1/notifier/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/util1/certs/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/tracking/tracking.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/supervisor/*.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/nginx/*.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/lms/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/cms/edx.log"
+        - "{{ COMMON_LOG_DIR }}/ecommerce/*"
+        - "{{ COMMON_LOG_DIR }}/ecommerce_worker/*"
+        - "{{ COMMON_LOG_DIR }}/lms/edx.log"
+        - "{{ logstash_log_dir }}/*.log"
+        - "{{ COMMON_LOG_DIR }}/mongo/*"
+        - "{{ COMMON_LOG_DIR }}/notifier/edx.log"
+        - "{{ COMMON_LOG_DIR }}/rabbitmq/*"
+        - "{{ COMMON_LOG_DIR }}/supervisor/*"
+
+      input_type: log
+
+output:
+
+  ### Logstash as output
+  logstash:
+    # The Logstash hosts
+    hosts: ["{{ LOGSTASH_HOST }}:{{ LOGSTASH_PORT }}"]
```

Comment:
Nitpick: We try to use multiline format for collections, where possible. It makes the code easier to extend/update in the future.

```
hosts:
  - "{{ LOGSTASH_HOST }}:{{ LOGSTASH_PORT }}"
```